commands/issue/view/issue_view.go

```diff
@@ -114,6 +116,13 @@ func NewCmdView(f *cmdutils.Factory) *cobra.Command {
 	return issueViewCmd
 }
 
+func openWeb(url string, browser string, io *iostreams.IOStreams) error {
```

Comment:
I'd prefer you make `io *iostreams.IOStream` the first parameter
```suggestion
func openWeb(io *iostreams.IOStreams, url string, browser string) error {
```

Reply:
Sure thing. I will fix this first thing tomorrow morning.